### ISSUE TITLE
DomainException when downloading from ilikecomix.com

### ISSUE THREAD

**Issue by PrivAccPriv** [jaymoulin/yamete#27]
With both Docker and PHP from cmd, I cannot download from ilikecomix.com. I have tried several commands, but they always result in a `[DomainException]` error. The Docker command I use is the following:
`docker run --rm -ti -v C:\Users\my_username\Downloads/app/downloads -u $(id -u) jaymoulin/yamete download -vu "ilikecomix.com/adultcomics/pet-girlfriend-3/" --ansi`

I have tried this command with `-u` instead of `-vu` as well, I have also tried several comics from ilikecomix.com, none seem to work. Removing `--ansi` also has no effect. The exact output is as follows:

```
Parsing ilikecomix.com/adultcomics/pet-girlfriend-3/

In DownloadCommand.php line 114:
  [DomainException]
  Unable to parse ilikecomix.com/adultcomics/pet-girlfriend-3/.
  Consider creating an issue on https://github.com/jaymoulin/yamete/issues/


Exception trace:
  at /app/src/DownloadCommand.php:114
 Yamete\DownloadCommand->execute() at /app/vendor/symfony/console/Command/Command.php:255
 Symfony\Component\Console\Command\Command->run() at /app/vendor/symfony/console/Application.php:1011
 Symfony\Component\Console\Application->doRunCommand() at /app/vendor/symfony/console/Application.php:272
 Symfony\Component\Console\Application->doRun() at
/app/vendor/symfony/console/Application.php:148
 Symfony\Component\Console\Application->run() at /app/download:32
```

I have tried many different things, but I cannot seem to fix it. Could it be a bug?

**Comment by jaymoulin** [comment]
Thank you for reporting this. If you like this product, please consider donation to help me continue maintain and create great products ([![PayPal donation](https://github.com/jaymoulin/jaymoulin.github.io/raw/master/ppl.png "PayPal donation")](https://www.paypal.me/jaymoulin) [![Buy me a coffee](https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png "Buy me a coffee")](https://www.buymeacoffee.com/3Yu8ajd7W) [![Become a Patron](https://badgen.net/badge/become/a%20patron/F96854 "Become a Patron")](https://patreon.com/jaymoulin))

Your issue is really valuable here for this kind of url is not handled yet by the driver.

P.S.: Please note you have to add the full url with its scheme (`https://DOMAIN.TLD`)

**Comment by jaymoulin** [comment]
This issue is available in latest public (`1.11.0`). `1.12.0` is released for Patreon supporters (https://www.patreon.com/posts/yamete-1-12-0-35510401)
